f2/src/locales/fr.json

```diff
@@ -22,10 +22,13 @@
   "banner.warning": "Attention! Ce site est un prototype. Votre rapport ne sera pas réellement soumis à la GRC.",
   "businessInfoPage.nextButton": "Continuer",
   "businessInfoPage.nextPage": "Étape suivante : Que s’est-il passé?",
-  "businessPage.business": "Comment votre entreprise pourrait-elle être affectée?",
-  "businessPage.businessExample": "Par exemple : actifs, réputation, clients",
+  "businessPage.industry": "Type d'industrie",
```

Comment:
Refer to the figma for translated content!
https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-9?node-id=3335%3A0

Reply:
fixed